src/highdicom/content.py

```diff
@@ -1341,7 +1347,13 @@ def __init__(
 
 class VOILUT(Dataset):
 
-    """Dataset describing a value-of-interest lookup table."""
+    """Class implementating the VOI LUT Macro.
+
+    Warning
+    -------
+    This class does **not** implement an item of the VOI LUT Sequence.
```

Comment:
I think technically speaking, it would still implement the macro. However, the macro attributes are included into the Frame VOI LUT Sequence:
https://dicom.nema.org/medical/dicom/current/output/chtml/part03/sect_C.7.6.16.2.html#table_C.7.6.16-11b

Even more confusingly, the VOI LUT macro includes the VOI LUT Sequence attribute and the `VOILUT` class has attribute `VOILUTSequence`. Therefore, it is important to make clear that instances of the class are not items of the VOI LUT Sequence attribute.

Reply:
Maybe we should rename the class to `VOILUTTransformation` or something along those lines. That would be a deviation from the standard, but it would probably be much less confusing.

Reply:
This discussion is no longer relevant (see https://github.com/herrmannlab/highdicom/pull/166#discussion_r855748603)